### DISCUSSION TITLE
including a novel model in pybamm

### DISCUSSION THREAD

**Discussion by salman-qadir** [pybamm-team/PyBaMM#4384] [General]
Hi

I am considering including novel models of  Li-ion batteries in the Pybamm. The introduction to the model is available in the following papers. PLEASE SUGGEST  is it worth doing? 
although the group has discussed it, I have seen no conclusive argument. 
 

https://www.researchgate.net/publication/332662933_Full_Homogenized_Macroscale_Model_and_Pseudo-2-Dimensional_Model_for_Lithium-Ion_Battery_Dynamics_Comparative_Analysis_Experimental_Verification_and_Sensitivity_Analysis

https://www.sciencedirect.com/science/article/abs/pii/S2352152X21014717

**Comment by valentinsulzer** [comment]
I would recommend you start by implementing it as a "standalone" model, following the instructions in https://github.com/pybamm-team/pybamm-cookie/tree/main

Let us know (@agriyakhetarpal @santacodes ) if you need any help.

Once you have implemented it and we can look at results / comparison to DFN, we can see if it makes sense to include it in main pybamm
